test/test.tf

```diff
@@ -0,0 +1,53 @@
+variable "vault_addr" {
```

Comment:
I would expect to see PR re https://github.com/aeternity/infrastructure/blob/491acdd2057328bd83051cc43bf3e9f01799acac/.circleci/config.yml#L213 alongside this. It looks like duplicated file. Which one is canonical?